tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -290,6 +291,7 @@ def _run(self):
                                 ],
                             )
                             rebal_det.save(ttl=SYNC_TTL)
+                            georep_details.save_georep_details(volumes, index)
```

Comment:
if This is a cluster operation, it should be only run by the gluster-integration with tag "provisioner/$integration_id"